File: semester 1/TugasWeek6/Lola Anggita Citrawilis_102022300151_TugasLooping.java
public class TugasLooping {
    public static void main(String[] args) {
        int n = 2;
        int count = 0;
        int jumlah = 0;
        int Maximum = Integer.MIN_VALUE;
        int Minimum = Integer.MAX_VALUE;

        while (n <= 15) {
            int angka = n;
            jumlah += angka;
            count++;
            n++;

            if (angka > Maximum) {
                Maximum = angka; 
            }

            if (angka < Minimum) {
                Minimum = angka; 
            }

        }

        double rataRata = (double) jumlah / count; // Hitung rata-rata (menggunakan pembagian floating point)

        System.out.println("Nilai Maksimum: " + Maximum);
        System.out.println("Nilai Minimum: " + Minimum);
        System.out.println("Rata-rata: " + rataRata);
    }
}

File: semester 1/TugasWeek6/Lola Anggita Citrawilis_102022300151_TugasLooping.py
angka = 2
total = 0
nilai_maksimum = angka
nilai_minimum = angka

while angka <= 15:
    total += angka
    
    if angka > nilai_maksimum:
        nilai_maksimum = angka
    
    if angka < nilai_minimum:
        nilai_minimum = angka
    
    angka += 1

rata_rata = total / 14 

# Menampilkan hasil
print("Nilai Maksimum:", nilai_maksimum)
print("Nilai Minimum:", nilai_minimum)
print("Rata-rata:", rata_rata)
